app/controllers/albums

```diff
@@ -0,0 +1,27 @@
+const rp = require('request-promise');
```

Comment:
`rp` is not expressive enough. Something like `request` would be easier understood.